### DISCUSSION TITLE
Can Anyone help me get some beginner badges :)

### DISCUSSION THREAD

**Discussion by HalemoGPA** [community/community#36686] [Other Feature Feedback, Questions, & Ideas]
Can Anyone help me get some beginner badges :) 
something like galaxy badge or pull shark

**Comment by johnmichaelarc** [comment]
This is the simplest word how to get Quick draw badge

(closed an issue / pull request within 5 minutes of opening)

Here is the list of all badge:-

https://github.com/drknzz/GitHub-Achievements
